### ISSUE TITLE
OpenSSL 1.0.2p

### ISSUE THREAD

**Issue by FeodorFitsner** [appveyor/ci#2558]

**Comment by RobertBColton** [comment]
This upgrade made it no longer possible to `vcpkg install grpc` on AppVeyor and has broken our build. I did not get any errors prior to this change and the build worked fine. Please see: https://github.com/Microsoft/vcpkg/issues/4189

**Comment by FeodorFitsner** [comment]
Why do you think the issue is related to OpenSSL update? Could you please elaborate?

**Comment by RobertBColton** [comment]
I ran my build that does `vcpkg install grpc` on AppVeyor with the same environment setup (Visual Studio 2017 image) just about an hour before you guys did the update. I thought I had the build perfect but then had to rerun it just after the update was deployed to AppVeyor and then the build failed. The only difference I could see in the log was the Visual Studio version.

Before/working: https://ci.appveyor.com/project/enigma-dev/radialgm-deps/build/1.0.57/job/ftn0ck64kpccv7mo#L4
After/not working (triggered by New Build UI button): https://ci.appveyor.com/project/enigma-dev/radialgm-deps/build/1.0.58/job/45cn3iff0u9ce8p1#L4

Here is the organization's repo and also I should mention that I just switched back to `image: Previous Visual Studio 2017` and the build went green.
https://github.com/enigma-dev/RadialGM-deps/commit/4155f5952df878bf5dd7b9d6edb9c79212ded6d2
https://ci.appveyor.com/project/enigma-dev/radialgm-deps/build/1.0.50/job/9sijsx5kgeb4b7qa#L4

So basically, I believe it's related to this issue by the process of elimination and the fact that I've observed this in a sort of controlled experiment way. I hope that you will look into this further, thank you.

**Comment by FeodorFitsner** [comment]
But why do you think it’s both VS 15.8.1 and OpenSSL updates broke the build? You posted the same comment in two issues.

**Comment by FeodorFitsner** [comment]
Looks like installing OpenSSL 1.1 regressed that: https://github.com/Microsoft/vcpkg/issues/4189#issuecomment-417462822

**Comment by RobertBColton** [comment]
Sorry about that but I wasn't really absolutely certain myself at that point which of the updates caused it and wanted to make sure to bring attention to it. I'm glad the issue has been rooted out and appreciate the quick turn around!
